dashboard/test/ui/features/javalab/neighborhood.feature

```diff
@@ -0,0 +1,22 @@
+@eyes
+  Feature: NeighborhoodPainting
+    
+Scenario: Paint Glomming Shapes
+  When I open my eyes to test "Paint Glomming Shapes"
+  Given I create a levelbuilder named "Simone"
+  And I am on "http://studio.code.org/s/allthethings/lessons/44/levels/7"
+  And I wait for the page to fully load
+  Then I press "runButton"
+  And I wait for 25 seconds
+  And I see no difference for "paint glomming"
+  Then I close my eyes
+
+#Scenario: Stop Button Closes Connection
```

Comment:
Did you mean to keep this commented out?

Reply:
I did- I explain in the PR description. I can remove it entirely, but figured since I already wrote it, the next person's task might be easier!

Reply:
Can you add a task to Jira and link it in this github comment thread to track the remaining work here?

Reply:
ahh sorry I missed that! I don't have any preference on removing it versus leaving it commented out so commented out seems good to me!

Reply:
https://codedotorg.atlassian.net/browse/CSA-624 This includes the work to create the console message (which @moneppo mentioned might differ based on mini-app). I will add a comment there pointing the engineer back to this file to finish this test!